src/extensions/renderer/base/coord-ele-math/coords.js

```diff
@@ -26,8 +26,9 @@ BRp.findContainerClientCoords = function(){
   }
 
   var container = this.container;
+  var containerWindow = container.ownerDocument.defaultView || window
```

Comment:
This sort of line is repeated often, so it would be a good candidate to have in a reusable function (e.g. `cy.window()`).  Since that function would only be used internally, it wouldn't have to be documented.